docs-v2/integrations/all/google-docs.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Google Docs
+sidebarTitle: Google Docs
+---
+
+API configuration: [`google-docs`](https://nango.dev/providers.yaml)
+
+## Features
+
+| Features | Status |
+| - | - |
+| [Auth (OAuth)](/integrate/guides/authorize-an-api) | ✅ |
+| [Sync data](/integrate/guides/sync-data-from-an-api) | ✅ |
+| [Perform workflows](/integrate/guides/perform-workflows-with-an-api) | ✅ |
+| [Proxy requests](/integrate/guides/proxy-requests-to-an-api) | ✅ |
+| [Receive webhooks](/integrate/guides/receive-webhooks-from-an-api) | 🚫 (time to contribute: &lt;48h) |
+
+<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+
+## Getting started
+
+-   [Google Docs access token specs](https://cloud.google.com/iam/docs/reference/sts/rest/v1/TopLevel/token#response-body)
+-   [Google Docs OAuth scopes](https://developers.google.com/docs/api/auth#docs-scopes)
```

Comment:
Should we add a link to the API reference?

Reply:
@TBonnin just updated my pull request to include the Google docs API reference.